roles/core/conman/tasks/main.yml

```diff
@@ -36,6 +36,16 @@
     state: present
     system: yes
 
+- name: user █ Ensure user conman can use /var/run/conman
+  copy:
+    content: |
+      D /var/run/conman 0775 conman conman -
+    dest: /usr/lib/tmpfiles.d/conman.conf
```

Comment:
I would prefer `/etc/tmpfiles.d/conman.conf` (see [Configuration Directories and Precedence](https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html#Configuration%20Directories%20and%20Precedence)), but are we the vendor or the local administrator here?
If there is a chance the upstream project (conman)  use this in a near future, usage of `/usr/lib/tmpfiles.d` may lead to some unexpected behavior.

Reply:
Sorry forgot this issue... I followed some RedHat blog page to design the role but I am not against your idea.